lib/quickbooks-payments/request.rb

```diff
@@ -17,6 +17,18 @@ class << self
 
         private
 
+        def root
+          if Quickbooks::Payments.sandbox_mode
+            'https://sandbox.api.intuit.com'
+          else
+            'https://api.intuit.com'
+          end
+        end
+
+        def get_url(relative_url = '')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.